From 482e64840986a2f4fa837bc651757314f8f72bd9 Mon Sep 17 00:00:00 2001
From: Albin B <albin.bala.krishnan@intel.com>
Date: Fri, 11 Dec 2015 16:07:20 +0530
Subject: [PATCH 3/3] [EM/PMIC_CCSM]: Fix device mode detection issue post PR
 swap

Cable type cant be determined after power role swap. This leads
to not sending the otg events for cable disconnect. So, device
mode cannot be detected further. Hence allowing to send
notifications to otg upon cable disconnect even though cable
type can't be determined.
Note: When Orig Change-Id: I4e612913313676da8e22243c289928970a979f91
is applied this change is missed.

Change-Id: Ie3c21c924b9481c9307fa6b0833a588d4e12823f
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-14772
Signed-off-by: Albin B <albin.bala.krishnan@intel.com>
Reviewed-on: https://android.intel.com:443/447406
Reviewed-on: https://android.intel.com:443/454502
---
 drivers/power/intel_pmic_ccsm.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/power/intel_pmic_ccsm.c b/drivers/power/intel_pmic_ccsm.c
index 409da9c..41c8acc 100644
--- a/drivers/power/intel_pmic_ccsm.c
+++ b/drivers/power/intel_pmic_ccsm.c
@@ -966,7 +966,6 @@ static void handle_internal_usbphy_notifications(int mask)
 	case POWER_SUPPLY_CHARGER_TYPE_NONE:
 	default:
 		cap.ma = 0;
-		chc.is_notify_otg = false;
 	}
 
 	dev_dbg(chc.dev, "Notifying OTG ev:%d, evt:%d, chrg_type:%d, mA:%d\n",
-- 
1.9.1

